<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_AmcReg</class>
 <widget class="QDialog" name="Dial_AmcReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Amc Registers Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_AmcReg_Write">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Register</string>
   </property>
   <widget class="QCheckBox" name="checkBox_AmcReg">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Hexadecimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Value</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Value</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_AmcReg_Read">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_AmcReg_Addr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Register Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_AmcReg_Cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AmcReg_Write">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_AmcReg_Read">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>371</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>lineEdit_AmcReg_Write</zorder>
  <zorder>pushButton_AmcReg_Cancel</zorder>
  <zorder>pushButton_AmcReg_Write</zorder>
  <zorder>pushButton_AmcReg_Read</zorder>
  <zorder>textEdit_Msg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
